web-api/terraform/template/elasticsearch.tf

```diff
@@ -49,3 +49,27 @@ resource "aws_elasticsearch_domain" "efcms-search" {
     log_type                 = "ES_APPLICATION_LOGS"
   }
 }
+
+resource "aws_elasticsearch_domain" "efcms-app-logs" {
+  domain_name           = "efcms-app-logs-${var.environment}"
+  elasticsearch_version = "7.4"
+
+  cluster_config {
+    instance_type = "t2.small.elasticsearch"
+    instance_count = var.es_instance_count == "" ? "1" : var.es_instance_count
+  }
+
+  ebs_options{
+    ebs_enabled = true
+    volume_size = 10
+  }
+
+  snapshot_options {
+    automated_snapshot_start_hour = 23
+  }
+
+  log_publishing_options {
+    cloudwatch_log_group_arn = aws_cloudwatch_log_group.elasticsearch_application_logs.arn
+    log_type                 = "ES_APPLICATION_LOGS"
+  }
+}
```

Comment:
I think we’ll also want to require HTTPS:

```hcl
resource "aws_elasticsearch_domain" "efcms-app-logs" {

  …

  domain_endpoint_options {
    enforce_https = true
    tls_security_policy = "Policy-Min-TLS-1-2-2019-07"
  }
}
```

Reply:
Nice! Terraform rocks